core/modrulestore_test.go

```diff
@@ -134,6 +138,8 @@ var _ = Describe("ModRuleStore", func() {
 		Entry("patch-10 on deployment-3 should work as expected", []string{"patch/patch-10.yaml"}, "deployment-3.json", "patch-10-deployment-3.txt"),
 		Entry("patch-11 on pod-6 should work as expected", []string{"patch/patch-11.yaml"}, "pod-6.json", "patch-11-pod-6.txt"),
 		Entry("patch-12 on pod-6 should work as expected", []string{"patch/patch-12.yaml"}, "pod-6.json", "patch-12-pod-6.txt"),
+		Entry("patch-13 on deployment-1 should work - ModRule in kubemod-system, but targetNamespaceRegex matches", []string{"patch/patch-13.yaml"}, "deployment-2.json", "patch-13-deployment-2.txt"),
+		Entry("patch-13b on deployment-1 should not work - ModRule in kubemod-system, but no targetNamespaceRegex", []string{"patch/patch-13b.yaml"}, "deployment-2.json", "empty-array.txt"),
```

Comment:
Please rename `patch-13b.yaml` to `patch-14.yaml`.

Reply:
👍